scripts/mbedtls_dev/bignum_core.py

```diff
@@ -759,12 +759,23 @@ class BignumCoreExpMod(BignumCoreTarget, bignum_common.ModOperationCommon):
     """Test cases for bignum core exponentiation."""
     symbol = "^"
     test_function = "mpi_core_exp_mod"
-    test_name = "Core modular exponentiation"
+    test_name = "Core modular exponentiation (Mongtomery form only)"
```

Comment:
Not a blocker, just a discussion comment.

Should we agree on a consistent wording for the representations? We need to keep on using Montgomery and Cannonical form on many places including tests and headers. 

And would there be value to have a abbreviated form for each?